core-backend/sandbox/src/funcs.rs

```diff
@@ -76,6 +73,23 @@ pub enum FuncError<E: Display> {
     ReadLenOverflow(u32, u32),
     #[display(fmt = "Binary code has wrong instrumentation")]
     WrongInstrumentation,
+    #[display(fmt = "Cannot decode value from memory")]
+    DecodeValueError,
+    #[display(fmt = "Buffer size {_0} is not eq to pre-registered size {_1}")]
```

Comment:
`is not equal ...`. What is the reason to shorten text in error/diagnostic messages?